src/hotspot/share/classfile/classListParser.cpp

```diff
@@ -460,19 +460,42 @@ bool ClassListParser::is_matching_cp_entry(constantPoolHandle &pool, int cp_inde
   }
   return true;
 }
-
 void ClassListParser::resolve_indy(Symbol* class_name_symbol, TRAPS) {
+  ClassListParser::resolve_indy_impl(class_name_symbol, THREAD);
+  if (HAS_PENDING_EXCEPTION) {
+    ResourceMark rm(THREAD);
+    tty->print("resolve_indy for class %s has", class_name_symbol->as_C_string());
```

Comment:
Shouldn't this be using appropriate unified logging?

Reply:
Ok. I've changed it to `LogTarget(Debug, cds, lambda)`.